### PR TITLE
fix: Signal installation in fedora

### LINKED ISSUE #808: Failed to Enable Signal Copr Repo
## Describe the bug
Installing Signal in Fedora fails, failing to enable the relevant copr repo.
It seems that the copr repo no longer exists, and linutil needs to be reconfigured to either use an alternative repo, or install the flatpak version.

## To Reproduce
1. Launch linutil on fedora
2. Go to "Applications Setup" > "Communication Apps"
3. Scroll down to "Signal"
4. Press Enter
5. Press "Y" and Enter to enable repo
6. See error

## Expected behavior
Signal client installs

## Screenshots
Screenshot from 2024-10-12 16-19-21

## Additional context
Ran on Fedora 40

## Checklist
- [x] I checked for duplicate issues.
- [x] I checked already existing discussions.
- [x] This issue is not included in the roadmap.
- [x] This issue is present on both stable and development branches.

### PR DESCRIPTION
**Pull request by jeevithakannan2**
## Type of Change
- [ ] New feature
- [x] Bug fix
- [ ] Documentation update
- [ ] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
- Replace repo with flatpak as it no longer exists.

## Issues / other PRs related
- Resolves #808 
- **Depends on** #705

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by Angxddeep** [comment]
This is one of the reason I asked for adding a flatpak option for applications.

File: core/tabs/applications-setup/communication-apps/signal-setup.sh [hunk 1]
```diff
@@ -20,8 +20,8 @@ installSignal() {
                 "$ESCALATION_TOOL" "$PACKAGER" -S --noconfirm signal-desktop
                 ;;
             dnf)
-                "$ESCALATION_TOOL" "$PACKAGER" copr enable luminoso/Signal-Desktop
-                "$ESCALATION_TOOL" "$PACKAGER" install -y signal-desktop 
+                checkFlatpak
```
Comment (leventbesli) [inline core/tabs/applications-setup/communication-apps/signal-setup.sh:23]: hi @jeevithakannan2, I wanted to check checkFlatpak but I can't see where it is pointing to. It's not in this file or common-script.sh. Am I missing something?

Comment (leventbesli) [reply core/tabs/applications-setup/communication-apps/signal-setup.sh:23]: nevermind. now i see it in other pr. :)

**Comment by adamperkowski** [comment]
Is this still relevant? With all the flatpak PRs we got here?

**Comment by adamperkowski** [comment]
> > Is this still relevant? With all the flatpak PRs we got here?
> 
> I have added the flatpak PR as depend in PR description

Wouldn't it just be easier to put this in the flatpak PR?

**Comment by jeevithakannan2** [comment]
> > > Is this still relevant? With all the flatpak PRs we got here?
> > 
> > I have added the flatpak PR as depend in PR description
> 
> Wouldn't it just be easier to put this in the flatpak PR?

Will do that.

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**